hunt/app/views/animals/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<ul>
+  <% @animals.each do |animal| %>
+    <!--
+      <li>
+        <% link_to animal.name, animal_path(animal)  %></li>
+      -->
```

Comment:
You can go ahead and delete this commented section, we'll add in links with the next task!